copyright.md

```diff
@@ -1,14 +1,12 @@
 # Open Systems Pharmacology Suite Manual
 
-Version 7.0
+Version 7.4
 
-Publication date February 2017
+Publication date February 2018
 
-Copyright © 2017 Open Systems Pharmacology Suite Community
+Copyright © 2018 Open Systems Pharmacology Suite Community
 
-License Owners of Open Systems Pharmacology Suite are allowed to print and use the PK-Sim® User Manual without fee, provided that the above copyright notice and this paragraph appear in all copies.
-
-The manual is provided “as is” without expressed or implied warranty. You may not modify it in any way.
+The manual is provided “as is” without expressed or implied warranty.
 
 **Trademark information:** All trademarks within this manual belong to their legitimate owners. PK-Sim® and MoBi® use the CVODE solver:
 
```

Comment:
Please change to "... use CVODE**S** solver..."

Reply:
Also think we should replace `PK-Sim® and MoBi® use the CVODES solver` with `Open Systems Pharmacology Suite uses the CVODES solver`